src/commands/functions-config-export.ts

```diff
@@ -0,0 +1,383 @@
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+import * as clc from "cli-color";
+import { flatten } from "flat";
+
+import * as env from "../functions/env";
+import * as functionsConfig from "../functionsConfig";
+import { Command } from "../command";
+import { FirebaseError } from "../error";
+import { testIamPermissions } from "../gcp/iam";
+import { logger } from "../logger";
+import { getProjectId } from "../projectUtils";
+import { promptOnce } from "../prompt";
+import { loadRC } from "../rc";
+import { requirePermissions } from "../requirePermissions";
+import { logBullet, logWarning, logSuccess } from "../utils";
+
+const REQUIRED_PERMISSIONS = [
+  "runtimeconfig.configs.list",
+  "runtimeconfig.configs.get",
+  "runtimeconfig.variables.list",
+  "runtimeconfig.variables.get",
+];
+
+const RESERVED_PROJECT_ALIAS = ["local"];
+
+interface Project {
```

Comment:
Maybe don't call this `Project` since that feels so big and canonical... `ProjectAliasInfo` or something maybe?